LiveSplit.AutoSplitters.xml

```diff
@@ -610,6 +610,7 @@
         </Games>
         <URLs>
             <URL>https://raw.githubusercontent.com/mitchell-merry/autosplitters/main/Pogo3D/pogo3d.asl</URL>
+            <URL>https://raw.githubusercontent.com/mitchell-merry/autosplitters/main/Pogo3D/Pogo3D.Settings.xml</URL>
```

Comment:
Won't this re-download the file every time you open LiveSplit, and overwrite the existing settings? Why not use LiveSplit's built in auto splitter settings?

Reply:
This is a file that is loaded to *provide* the built-in auto splitter settings.

Reply:
Do you have to modify this file later to change the settings? For an ASL script, LiveSplit re-downloads all the files every time, so it would re-download the settings file as well

Reply:
You would modify this file to change the settings, yes. It doesn't have any effect on the settings stored in any split files, if that's a concern.

Reply:
Yeah, so you'd activate the autosplitter, which downloads the settings file. Then you'd modify the settings file to change the settings. But when you re-open LiveSplit, it would re-download the settings file and overwrite your changes

Reply:
I'm sorry, I'm not understanding what you're getting at here. This file is not to be modified by users, just like the ASL file is not to be modified by users.

Reply:
Oh okay, I misunderstood what the purpose of the settings file is, then

Reply:
Yep exactly, it just holds the definitions for the settings rather than any information about what settings the user currently has. That's handled by livesplit and the settings window as normal. 


Could I get a merge on this?